.travis.yml

```diff
@@ -1,7 +1,29 @@
+sudo: required
+
 language: node_js
 node_js:
-  - "stable"
+- stable
+
+services:
+- docker
+
+before_install:
+- docker build -t kevinchu/first-blog-mern .
+- docker run -d -p 127.0.0.1:4000:3000 kevinchu/first-blog-mern
+- docker ps -a
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
Yes, yaml optionally allows spaces before the hyphen for arrays, but you should be consistent

Reply:
fixed